buf-kvutils.yaml

```diff
@@ -21,12 +17,8 @@ build:
     - daml-lf/archive/src/main/protobuf/com/daml/daml_lf_1_14/
     - daml-lf/archive/src/main/protobuf/com/digitalasset/
 
-lint:
-  use:
-    - DEFAULT
-  except:
-    - PACKAGE_VERSION_SUFFIX
-
 breaking:
   use:
-    - WIRE_JSON
+    - WIRE
```

Comment:
@miklos-da : can you explain your rationale for `WIRE` instead of `WIRE_JSON`. Ideally, we'd add that rationale as a comment here so future developers will be able to build on it.

Reply:
Proposed a wording in df65d20.

Reply:
Saying that we need to be able to read data persisted by previous versions of the code should be enough. I.e., I don't think we need to elaborate, as the previous implies we don't care about field names just the semantics of the data. We should never persist data in JSON format from kvutils.

Reply:
370adc2